src/modules/legacy/components/Summary/summary.css

```diff
@@ -1,7 +1,7 @@
 .container {
   & > header {
-    padding-left: var(--horizontal-padding-l) !important;
-    padding-right: var(--horizontal-padding-l) !important;
+    padding-left: calc(var(--horizontal-padding-l) / 2) !important;
+    padding-right: calc(var(--horizontal-padding-l) / 2) !important;
```

Comment:
Do you mean to use `--horizontal-padding-s`?

Reply:
@reyraa Fixed